monsteer/CMakeLists.txt

```diff
@@ -18,5 +18,5 @@ set(MONSTEER_LINK_LIBRARIES PUBLIC Brion Lunchbox zeq)
 
 common_library(Monsteer)
 
+add_subdirectory(brionplugin)
```

Comment:
In Fivox it's called just livre, not livreplugin (same for the namespace). Obviously I prefer the shorter one. ;)

We can also go the long name, but in any case we should keep it consistent.